tests/verify-size.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
All of these tasks should probably be conditional on `device_type` being "lvm".

Reply:
I specified the device type as 'lvm' in 'verify-main' when including the tasks for 'verify-size'.

Reply:
If verify-size.yml is lvm-specific it should be named verify-lvm-size.yml.